### PR TITLE
Better logging for retries

### PR DESCRIPTION
**Pull request by lstrojny**
This PR introduces better logging for redis-solr retries.

### CODE REVIEW

File: src/main/java/com/sematext/solr/redis/RetryingCommandHandler.java [hunk 1]
```diff
@@ -65,7 +71,17 @@ public RetryingCommandHandler(final JedisPool connectionPool, final int maxRetri
...
+      log.info("Redis communication error recovered after {}ms and {} of {} retries for command {} on {}",
+          currentTimeMillis() - methodInvocationTime, retries, maxRetries, command.getClass().getName(),
+          localParams.get("key"));
+    }
+
+    return results;
+  }
+
+    /** @return Current time in milliseconds */
+  private static long currentTimeMillis() {
```
Comment (otisg) [inline src/main/java/com/sematext/solr/redis/RetryingCommandHandler.java]: Isn't this redundant?  Why not call System.currentTimeMillis() directly?

Comment (lstrojny) [reply src/main/java/com/sematext/solr/redis/RetryingCommandHandler.java]: Wanted to keep it DRY but I am not married to it 😄

**Comment by lstrojny** [comment]
Tested the patch now, seems to work fine. Ready to be merged from my POV.

File: src/main/java/com/sematext/solr/redis/RetryingCommandHandler.java [hunk 2]
```diff
@@ -73,7 +73,7 @@ public RetryingCommandHandler(final JedisPool connectionPool, final int maxRetri
 
     if (retries > 0) {
       log.warn("Redis communication error recovered after {}ms and {} of {} retries for command {} on {}",
-          currentTimeMillis() - methodInvocationStart, retries, maxRetries, command.getClass().getName(),
+          currentTimeMillis() - methodInvocationStart, retries + 1, maxRetries, command.getClass().getName(),
```
Comment (otisg) [inline src/main/java/com/sematext/solr/redis/RetryingCommandHandler.java]: Why not just call System.currentTimeMillis() here?

Comment (lstrojny) [reply src/main/java/com/sematext/solr/redis/RetryingCommandHandler.java]: See above. Don't mind changing it though

**Pull request merged**